domehunter/gRPC-server/hx2dome.proto_server.py

```diff
@@ -571,7 +573,25 @@ def serve(kwargs_dict):
                         to the TheSkyX.")
     parser.set_defaults(server_testing=False)
 
-    kwargs_dict = vars(parser.parse_args())
+    parser.add_argument('-y', '--yamlconfig',
```

Comment:
I would just keep this as config, which is more standard.
```suggestion
    parser.add_argument('-c', '--config',
```

Reply:
I was already using a `-c` flag for another argument